hnn_core/params.py

```diff
@@ -256,6 +261,9 @@ def _extract_drive_specs_from_hnn_params(params, cellname_list):
             for cellname in cellname_list:
                 if cellname in par:
                     rate_params[cellname] = par[cellname][3]
+                    # correct for non-positive poisson rate constant
```

Comment:
is this a hack? Add XXX ? The `rate_params` is not used later? raise warning?

Reply:
Correct, this is a hack that prevents [`_check_poisson_rates()`](https://github.com/jonescompneurolab/hnn-core/blob/40b07726a747c336debfe3a2683ee1607391a75d/hnn_core/drives.py#L60) from throwing an error and excluding the null poisson drive specified in `default.json`.